commons-hpcc/pom.xml

```diff
@@ -85,11 +85,11 @@
             <artifactId>json</artifactId>
             <version>20180813</version>
         </dependency>
-    	<dependency>
-    		<groupId>org.apache.logging.log4j</groupId>
-    		<artifactId>log4j-api</artifactId>
-    		<version>2.10.0</version>
-		</dependency>
+        <dependency>
```

Comment:
Good catch, this was actually fixed in candidate-7.2.6, but will show up in master soon. You might need to rebase in order to avoid merge conflicts.